tests/database/test_eth1_chaindb.py

```diff
@@ -194,3 +214,80 @@ def test_chaindb_get_receipt_by_index(
             NUMBER_BLOCKS_IN_CHAIN,
             TRANSACTIONS_IN_BLOCK + 1,
         )
+
+
+@pytest.mark.parametrize(
+    ("use_persist_unexecuted_block"),
```

Comment:
*super-nit* why the extra parentheses here?

Reply:
yeah, copy pasta leftover